types/office-js-preview/index.d.ts

```diff
@@ -18440,17 +18496,17 @@ declare namespace Office {
          *
          * @param options - An object literal that contains one or more of the following properties:-
          *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
-         * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter of
-         *                 type `Office.AsyncResult`.
+         * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter, `asyncResult`,
+         *                 which is an `Office.AsyncResult` object. The appointment's sensitivity level is returned in the `asyncResult.value` property.
          *
          * @beta
          */
         getAsync(options: Office.AsyncContextOptions, callback: (asyncResult: Office.AsyncResult<MailboxEnums.AppointmentSensitivityType>) => void): void;
```

Comment:
It's curious to me that this isn't called `getLevelAsync`. Are we that certain sensitivity won't be expanded in the future?